azure/services/managedclusters/spec.go

```diff
@@ -491,6 +559,49 @@ func (s *ManagedClusterSpec) Parameters(ctx context.Context, existing *asocontai
 		}
 	}
 
+	if s.SecurityProfile != nil {
```

Comment:
Could you please update this particular unit test with the new fields?
https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/fd9cefa58bc9f95dc2ea3ee6a041ed3fc71649b2/azure/services/managedclusters/spec_test.go#L36

The other subtest there doesn't need to be updated.